docs/tab-widgets/distributed-trace-receive.asciidoc

```diff
@@ -21,7 +21,7 @@ traceContext.State, _ = apmhttp.ParseTracestateHeader(r.Header["Tracestate"]...)
 opts := apm.TransactionOptions{
 	TraceContext: traceContext, <3>
 }
-transaction := apm.DefaultTracer.StartTransactionOptions("GET /", "request", opts) <4>
+transaction := apm.DefaultTracer().StartTransactionOptions("GET /", "request", opts) <4>
```

Comment:
@carsonip we should also update the godoc.org links above to pkg.go.dev, and reference the /v2 import paths